src/main/kotlin/no/nav/syfo/tilgang/TilgangService.kt

```diff
@@ -34,8 +37,30 @@ class TilgangService(
         return tilgang
     }
 
+    suspend fun hasTilgangToEnhet(token: Token, callId: String, enhet: Enhet): Tilgang {
```

Comment:
Her er det `hadTilgangToEnhet`, mens der oppe er det `hasTilgangTilSyfo`, så stemmer for at vi holder oss til enten **til** eller **to**. Personlig holder jeg en knapp på **to** 🇬🇧